src/Common/src/System/Drawing/ColorTranslator.cs

```diff
@@ -109,88 +122,86 @@ public static int ToOle(Color c)
                 }
             }
 
-            return ToWin32(c);
+            return c.R << COLORREF_RedShift | c.G << COLORREF_GreenShift | c.B << COLORREF_BlueShift;
```

Comment:
Why can't we just call `ToWin32(c)`? It does the exact same thing.

Reply:
My mistake- I missed undoing that when I was iterating. :)